support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -139,4 +139,28 @@ export const tests: Tests = {
 			'^/uk/(checkout|thank-you)',
 		excludeContributionsOnlyCountries: true,
 	},
+	similarProductsConsent: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'VariantA',
+			},
+			{
+				id: 'VariantB',
+			},
+		],
+		audiences: {
+			GBPCountries: {
```

Comment:
Presume ConsentOrPay is a UK requirement only?

Reply:
Oh this should be global, good spot